deflate.c

```diff
@@ -593,7 +593,7 @@ int ZEXPORT PREFIX(deflatePrime)(PREFIX3(stream) *strm, int bits, int value) {
         put = BIT_BUF_SIZE - s->bi_valid;
         if (put > bits)
             put = bits;
-        s->bi_buf |= (uint64_t)((value & ((1 << put) - 1)) << s->bi_valid);
+        s->bi_buf |= (((uint64_t)value & (((uint64_t)1 << put) - 1)) << s->bi_valid);
```

Comment:
For constants, you should use UINT64_C() macro instead...

Reply:
Fixed. Thanks!